File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Options/IOptionData.php
<?php

namespace Ohtap\ECommerce\DomainModels\Objects\Options;


use  Ohtap\ECommerce\DomainModels\Core\Object as  CoreObjects ;


interface IOptionData extends CoreObjects\IObjectID, CoreObjects\IObjectCode,
    CoreObjects\IObjectDescription, CoreObjects\IObjectName, CoreObjects\IObjectOrder, CoreObjects\IObjectState, CoreObjects\IObjectTitle
{

}

File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Models Objects/Options/IOptionData.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CoreObject = Ohtap.ECommerce.Domain.Models.Core.Object;

namespace Ohtap.ECommerce.Domain.Models.Objects.Options
{
    public interface IOptionData :  CoreObject.IObjectID, CoreObject.IObjectCode, CoreObject.IObjectName, CoreObject.IObjectOrder, CoreObject.IObjectDescription, CoreObject.IObjectState
    {

    }
}
